server/Arcadia.Assistant.Web/Controllers/DownloadWebController.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Arcadia.Assistant.Web.Controllers
+{
+    using System.IO;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using Microsoft.AspNetCore.Mvc;
+
+    using Akka.Actor;
+
+    using Arcadia.Assistant.Server.Interop;
+    using Arcadia.Assistant.Web.Configuration;
+    using Arcadia.Assistant.Web.Download;
+
+    [Route("/download")]
+    public class DownloadWebController : Controller
+    {
+        private readonly IActorRefFactory actorSystem;
+        private readonly ITimeoutSettings timeoutSettings;
+
+        public DownloadWebController(
+            IActorRefFactory actorSystem,
+            ITimeoutSettings timeoutSettings)
+        {
+            this.actorSystem = actorSystem;
+            this.timeoutSettings = timeoutSettings;
+        }
+
+        [HttpGet]
+        public IActionResult Index()
+        {
+            return this.View();
+        }
+
+        [Route("get/{applicationType}")]
+        [HttpGet]
+        public async Task<IActionResult> GetFile(ApplicationType applicationType, CancellationToken token)
+        {
+            var downloadActor = this.actorSystem.ActorSelection(
+                $"/user/{WellKnownActorPaths.DownloadApplicationBuilds}");
+
+            var getBuildApplicationType = applicationType == ApplicationType.Android
+                ? GetLatestApplicationBuildPath.ApplicationTypeEnum.Android
+                : GetLatestApplicationBuildPath.ApplicationTypeEnum.Ios;
```

Comment:
400 else?

Reply:
btw, can you do the same as in PushNotifications and have enum in a type? will it throw 400 by default?

Reply:
hello?

Reply:
Done.